app/views/conferences/edit_notifications.html.haml

```diff
@@ -17,7 +17,7 @@
   %h1 Edit
 
   .row
-    .span6.offset8.label
+    .span6.offset8.label{ :style => "text-transform: none !important" }
```

Comment:
I think that the style is really specific to this particular element and I wondered why bootstrap would force a text-transform in the first place. I also wanted to make sure that no more specific rule coming from another unexpected source would ever mess with the display in this element where the actual representation of the content is crucial.

Having said this, I generally agree with your remark and if you could point me to where to fix this in an acceptable way, I'd love to implement it.